### PR TITLE
docs: upgrade by migrating to a new server

### PR DESCRIPTION
**Pull request by jooola**

### CODE REVIEW

File: docs/admin-manual/setup/upgrade.md [hunk 1]
```diff
@@ -39,10 +45,22 @@ You might need to run steps before and after the install procedure. Be sure to f
 
 :::
 
+:::caution
+
+If you are migrating LibreTime to a new server, you should read the upgrade procedure but must not apply them.
```
Comment (paddatrapper) [inline docs/admin-manual/setup/upgrade.md:50]: May add something about when to apply them?

Comment (jooola) [reply docs/admin-manual/setup/upgrade.md]: This is already defined few lines above in the "normal" upgrade procedure (when we don't migrate to a new server):

image

**Comment by jooola** [comment]
I re targeted this to the stable branch.

File: docs/admin-manual/setup/upgrade.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ You should always have proper backups and a rollback scenario in place before up
 
 :::
 
+:::info
+
+You may upgrade LibreTime by migrating to a new server. This is probably the cleanest way if you are coming from [Airtime](./migrate-from-airtime.md), if you wish to upgrade the underlying system, or if you want to skip multiple upgrade procedure.
```
Comment (paddatrapper) [inline docs/admin-manual/setup/upgrade.md:16]: Procedures plural?

Comment (jooola) [reply docs/admin-manual/setup/upgrade.md]: Solved

**Pull request merged**